handlers/download.go

```diff
@@ -166,7 +165,9 @@ func (d Download) Do(extension string) http.HandlerFunc {
 					Key:    &filename,
 				}
 
-				pskStr, err := d.VaultClient.ReadKey(d.VaultPath, filename)
+				vaultPath := d.VaultPath + "/" + filename
```

Comment:
Would it be clearer to rename `filename` to `dirname` as the vaultKey is really the filename? Or have I misunderstood?